src/main/java/org/apache/commons/imaging/color/ColorConversions.java

```diff
@@ -643,28 +602,30 @@ private static int convertRGBtoRGB(int red, int green, int blue) {
         return rgb;
     }
 
+    private static int convertRGBtoRGB(final Color color) {
+        return color.getRGB() | 0xff << 24; // alpha channel always opaque
+    }
```

Comment:
Adding the direct use of java.awt.Color.

Reply:
Ideally we would limit awt use, or remove it, so that users can use Imaging for Android apps too (we have one user with a workaround on his code at the moment). Unless that makes maintaining the code extremely difficult, in which case we can just try to figure out another way to provide a version compatible with Android...

Reply:
Reason taken. I will remove the java.awt reference.